ensembl/htdocs/info/genome/variation/species/species_data_types.html

```diff
@@ -736,8 +736,18 @@ <h3><a href="https://www.lovd.nl">LOVD</a></h3>
 
 <h3><a href="https://decipher.sanger.ac.uk">DECIPHER</a></h3>
 <p>
-We display both structural variants and short variants from DECIPHER. Data are provided on GRCh37 and remapped by Ensembl to GRCh38. These data are only available for display in the Browser, and not for bulk download.<br />
+We display both structural variants and short variants from DECIPHER. Data provided are only available for display in the Browser, and not for bulk download.<br />
 Access to bulk data may be obtained directly from DECIPHER (<a href="https://decipher.sanger.ac.uk/datasharing/">https://decipher.sanger.ac.uk/datasharing/</a>) and is subject to a Data Access Agreement, in which the user certifies that no attempt to identify individual patients will be undertaken. The same restrictions apply to the public data displayed in this browser no one is authorized to attempt to identify patients by any means.</p>
 
+
+<br />
+<h2 id="other_data">Other data</h2>
+
+<h3><a href="https://www.genomenon.com/mastermind">Mastermind</a></h3>
+<p>
+We display short variants from the Genomenon Mastermind Cited Variants Reference (<a href="https://www.genomenon.com/cvr/">https://www.genomenon.com/cvr/</a>). We use the Mastermind GRCh37 HGVS as the variant name for both GRCh37 and GRCh38 assemblies. <br />
+When a variant has more than one alternative allele, note that only the first alt allele has the HGVS displayed and linked out to Mastermind.
```

Comment:
`alt` feels a little clunky, I suggest:

`If a variant has multiple alternative alleles, please note that Mastermind and HGVS output will only be displayed for one allele.`